gallery/plot_transforms_v2_e2e.py

```diff
@@ -1,4 +1,6 @@
 """
+.. _e2e_object_detection_v2:
```

Comment:
Nit: can we align this label with the file name

```suggestion
.. _transforms_v2_e2e:
```

Reply:
You don't need this, you can refer to sphinx-gallery example with e.g. ` :ref:`sphx_glr_auto_examples_plot_transforms_v2_e2e.py`.`

Reply:
Even better